src/vpart_range.h

```diff
@@ -84,8 +84,8 @@ namespace std
 template<class T> struct iterator_traits<vehicle_part_iterator<T>> {
     using difference_type = size_t;
     using value_type = vpart_reference;
-    // TODO: maybe change into random access iterator? This requires adding
-    // more operators to the iterator, which may not be efficient.
```

Comment:
Why removing this comment? The iterator is still non-random-access as far as I can tell, so we should keep the "change into random access iterator" part.